### ISSUE TITLE
Optimize for mobile and make it a flexible layout

### ISSUE THREAD

**Issue by joshghent** [joshghent/alexswonderlandpuzzles#3]
Optimize the layout so that it is able to be rendered on mobile, currently it gets formatted weirdly on small screens

**Comment by readmey** [comment]
Is this issue still valid? If yes, can I work on this?

**Comment by joshghent** [comment]
I believe this issue is pretty much finished! but there is content on the page that isn't displayed so you could work on showing that information? https://github.com/joshghent/alexswonderlandpuzzles/blob/master/src/App.js#L90
Alternatively, another issue you could work on is #1

**Comment by readmey** [comment]
@joshghent sure i can do it !
you having any preferences how the "about" content should looks like?

**Comment by joshghent** [comment]
@readmey Not really! Just similar to how the existing content looks 👍

**Comment by readmey** [comment]
just  added a PR https://github.com/joshghent/alexswonderlandpuzzles/pull/14
What about using SCSS or extracting the form on the homepage into an own component?

**Comment by joshghent** [comment]
@readmey Yeah that is a very good idea! Happy for you to do that in a new PR if you like! 👍
